frontend/src/app/main/data/workspace.cljs

```diff
@@ -43,6 +43,7 @@
    [clojure.set :as set]
    [clojure.set :as set]
    [cuerdas.core :as str]
+   [cljs.pprint :refer [pprint]]
```

Comment:
Remove this import

Reply:
I thought about deleting it, but down later there are several commented-out pprints. I think they should be converted to using the new logging framework. But for the moment the import is not unneeded, also commented out.